Source/Fuse.Common/NameValuePair.uno

```diff
@@ -8,10 +8,13 @@ namespace Fuse
 		Implements `IObject`, which means the `NameValuePair` can be viewed as an object
 		with a single property.
 
+		Implements `IArray`, which means the `NameValuePair` can be viewed as an array
+		with a single element.
+
 		An `IArray` containing some `NameValuePairs` can be converted to an `IObject`
 		implementation containing all those properties using the `ObjectFromArray` method.
 	*/
-	public sealed class NameValuePair : IObject
+	public sealed class NameValuePair : IObject, IArray
```

Comment:
In particular, if a property takes an `IArray` and a NVP is provided the resulting array *must* be the NVP, not just the value.  So I think this is wrong here to make `NameValuePair` implement `IArray`.